samples/cmaf/ham/cmaf-ham-conversion/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "cmaf-ham-conversion",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "dev": "node --loader ts-node/esm src/index.ts",
+        "serve": "serve --cors ./output"
+    },
+    "dependencies": {
+        "@svta/common-media-library": "file:../../../../../../lib/",
+        "fs": "^0.0.1-security",
+        "ts-node": "^10.9.2",
+        "tsc": "^2.0.4"
```

Comment:
These dependencies aren't needed as they are already installed in the root package file.